.tekton/push.yaml

```diff
@@ -0,0 +1,50 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+metadata:
+  name: pipeline-service-push
+  annotations:
+    pipelinesascode.tekton.dev/on-event: "push"
+    pipelinesascode.tekton.dev/on-target-branch: "main"
+    pipelinesascode.tekton.dev/max-keep-runs: "5"
+spec:
+  params:
+    - name: git-url
+      value: "{{ repo_url }}"
+    - name: revision
+      value: "{{ revision }}"
+  pipelineSpec:
+    params:
+      - description: 'Source Repository URL'
+        name: git-url
+        type: string
+        default: "https://github.com/openshift-pipelines/pipeline-service"
+      - description: 'Revision of the Source Repository'
+        name: revision
+        type: string
+        default: "7e0ac52f0888d8d2f595152cdd44a09c58f5be37"
+    tasks:
+      - name: update-infra-repo
+        params:
+          - name: ORIGIN_REPO
+            value: $(params.git-url)
+          - name: REVISION
+            value: $(params.revision)
+          - name: SCRIPT
+            value: |
+              sed -i -E 's/[0-9a-f]{40}/$(params.revision)/g' components/pipeline-service/development/kustomization.yaml
+              sed -i -E 's/[0-9a-f]{40}/$(params.revision)/g' components/pipeline-service/staging/base/kustomization.yaml
```

Comment:
this regular expression should leave https://github.com/redhat-appstudio/infra-deployments/blob/main/components/pipeline-service/staging/base/kustomization.yaml#L11 alone until that ref is converted to a main branch commit sha